src/main/java/com/cepengagementservice/Services/UserServices.java

```diff
@@ -45,11 +64,19 @@ public boolean check(int userId) {
             return false;
     }
 
-    
+    /**
+     * basic method to find by email
+     * @param email
+     * @return a user
+     */
```

Comment:
Mention that is is finding a user by e-mail, and what it might return if it the e-mail doesn't come up (null or exception?)